tools/toktx/toktx.cc

```diff
@@ -732,9 +732,15 @@ toktxApp::main(int argc, _TCHAR *argv[])
                                     options.assign_oetf == KHR_DF_TRANSFER_UNSPECIFIED,
                                     rescale);
 
-            if (options.astc && image->getComponentSize() == 2)
-                options.astcopts.mode = KTX_PACK_ASTC_ENCODER_MODE_HDR;
+            // If input is > 8bit and user wants LDR issue quality loss warning
+            if (options.astc && image->getComponentSize() == 2
+                && options.astcopts.mode == KTX_PACK_ASTC_ENCODER_MODE_LDR) {
+                cerr << name << ": Warning! input file is 16bit but LDR option is provided."
```

Comment:
Say "specified" not "provided" and in line 746 below as well.

The commit message for this change suggests that LDR could be the default even when input is > 8 bits. Please make sure that the default for > 8-bits is HDR.

Reply:
>  Please make sure that the default for > 8-bits is HDR.

I was thinking about this. its a bit problematic. We are already issuing a warning that user has either provided LDR mode or using the "default" LDR mode. If we are going to chose HDR mode for > 8 bit then this warning shouldn't be there. It will go away. 

Basically my original two options:
3. If input is >8 bit and user wants LDR, warn of quality loss but continue
4. In no LDR/HDR option provided, for <=8 bit default LDR output otherwise default HDR output
Are mutually exclusive.

Reply:
I don't see how they are mutually exclusive. In no. 3 the user is explicitly specifying LDR. In no. 4 the user is not specifying any preference. I think this is fine.

Reply:
> Say "specified" not "provided" and in line 746 below as well.

Fixed in https://github.com/KhronosGroup/KTX-Software/pull/433/commits/49f793bdc50a175812c122c8a6fe2b5d113426a2 

Regarding the issue with 3 and 4. Ok not mutually exclusive but for option 4 we don't know if no options were provided. The way the arg persers and the default options work was that if nothing is provided we just default to something and by this time in `toktx.cc` we don't know if the option is the default or provided by the user.  So I have now added a default option as well which is then changed at `toktx.cc` scope. Fix is https://github.com/KhronosGroup/KTX-Software/pull/433/commits/8860912e8134e6a6d2f027b4d1da2748f426ec45